proposals/list-patterns.md

```diff
@@ -81,16 +81,16 @@ The behavior of a pattern-matching operation is undefined if any of the above as
 A pattern of the form `expr is [1, 2, 3]` is equivalent to the following code (if compatible via implicit `Index` support):
 ```cs
 expr.Length is 3
-&& expr[0] is 1
-&& expr[1] is 2
-&& expr[2] is 3
+&& expr[new Index(0, fromEnd: false)] is 1
+&& expr[new Index(1, fromEnd: false)] is 2
+&& expr[new Index(2, fromEnd: false)] is 3
```

Comment:
> (if compatible via implicit `Index` support)

This section presents the lowering with implicit indexer (using an int arg)

Reply:
> This section presents the lowering with implicit indexer (using an int arg)

Removed the reference to the implicit indexer there.